Security/SessionDownloadStrategy.php

```diff
@@ -39,14 +47,25 @@ class SessionDownloadStrategy implements DownloadStrategyInterface
     protected $sessionKey = 'sonata/media/session/times';
 
     /**
-     * @param TranslatorInterface $translator
-     * @param ContainerInterface  $container
-     * @param int                 $times
+     * @param TranslatorInterface        $translator
+     * @param ContainerInterface|Session $container
+     * @param int                        $times
      */
-    public function __construct(TranslatorInterface $translator, ContainerInterface $container, $times)
+    public function __construct(TranslatorInterface $translator, $container, $times)
```

Comment:
> but if you extend it the signature is different...

@OskarStark Argument names do no matter at all : https://3v4l.org/MAgfG , because different names don't make the signatures incompatible (both object can still be used with the same arguments)